Mogakco/Sources/Presentation/Profile/ViewModel/EditProfiileViewModel.swift

```diff
@@ -104,15 +114,26 @@ final class EditProfiileViewModel: ViewModel {
         let profileCreate = input
             .completeButtonTapped
             .withLatestFrom(type)
-            .filter { $0 == .create }
+            .filter {
+                if case EditType.create = $0 {
+                    return true
+                }
+                return false
+            }
```

Comment:
```suggestion
            .filter { $0 == .create }
```
== 연산자 정의되었으면 위 코드 동작하지 않나요?!

Reply:
저도 될 줄 알았는데 안되더라구요 😭

<img width="916" alt="스크린샷 2022-11-23 오후 5 58 29" src="https://user-images.githubusercontent.com/109145755/203506172-043b367a-ba78-43bc-8ca5-6ce5827b5ea7.png">

Reply:
이런.. 저 연관값도 더 깔끔하게 쓰는법을 좀 찾아봐야겠네요! 친절한 설명 감사합니다!